src/Context/Auth-context/Auth-context.js

```diff
@@ -28,24 +28,22 @@ const AuthProvider = ({ children }) => {
   };
 
   const loginHandler = async ({ email, password }) => {
-    if (isAuth) {
-      try {
-        const response = await axios.post("/api/auth/login", {
-          email,
-          password,
-        });
+    console.log("data");
```

Comment:
remove console. if there is no need further

Reply:
ok sure i will do this